OuterWildsRandomSpeedrun/SpeedrunHarmonyPatches.cs

```diff
@@ -52,4 +53,23 @@ public static void DebugInputManager_Start_Postfix()
   {
     SpeedrunState.IsGameStarted = true;
   }
+
+  // Disable achievements while the mod is running, since many warps can accidentally
+  // trigger ones like Hotshot.
+  [HarmonyPostfix]
+  [HarmonyPatch(typeof(Achievements), nameof(Achievements.Earn))]
+  public static bool Achievements_Earn_Postfix(Achievements.Type type)
+  {
+    var modHelper = OuterWildsRandomSpeedrun.OuterWildsRandomSpeedrun.Instance.ModHelper;
+    if (SpeedrunState.ModEnabled)
+    {
+      modHelper.Console.WriteLine($"Skipping achievement {type} during Nomai Grand Prix run.", MessageType.Info);
+      return false;
```

Comment:
Since this is a postfix patch, the normal code of Achievements.Earn is going to run before executing this, so the achievement will still be awarded

Reply:
wasn't thinking and copied an example from earlier in the file, mb. fixing.